client/overview/task-list/tasks.js

```diff
@@ -77,14 +77,18 @@ export const getTasks = ( {
 				),
 				additionalInfo: accountDetailsTaskDescription,
```

Comment:
I think we should change this to `content` given it generates a bigger gap between the title and the extra text, see screenshot:
<img width="1022" alt="Screen Shot 2021-09-23 at 11 41 47 AM" src="https://user-images.githubusercontent.com/2240960/134528927-0d52bfed-731e-4ace-9229-76aa6da737bc.png">

Reply:
Nice catch! I addressed that change in the commit aed0044